framework/src/main/java/org/checkerframework/framework/source/SourceChecker.java

```diff
@@ -323,6 +323,10 @@
     // org.checkerframework.framework.util.typeinference.DefaultTypeArgumentInference
     "showInferenceSteps",
 
+    // Output stack trace when checker encounters errors
+    // org.checkerframework.common.basetype.BaseTypeChecker.printStackTrace()
+    "dumpOnErrors",
```

Comment:
You should also add documentation to the manual.
There is already a comment on `doe` https://github.com/typetools/checker-framework/blob/d11aeaedaf4c039beb648ee0e3c177f894c060df/docs/manual/introduction.tex#L835
There you should mention this option.

You also need to document the feature itself, here in the [introduction](https://github.com/typetools/checker-framework/blob/d11aeaedaf4c039beb648ee0e3c177f894c060df/docs/manual/introduction.tex#L748) (find the appropriate place there) and in the corresponding section.

Reply:
I added the documentation in the introduction.tex file in the debugging section.